packages/app/src/components/DBTimeChart.tsx

```diff
@@ -149,10 +149,20 @@ export function DBTimeChart({
     const to = add(clickedActiveLabelDate, {
       seconds: convertGranularityToSeconds(granularity),
     }).getTime();
+    let where = config.where;
+    let whereLanguage = config.whereLanguage || 'lucene';
+    if (
+      where.length === 0 &&
+      Array.isArray(config.select) &&
+      config.select.length === 1
+    ) {
+      where = config.select[0].aggCondition ?? '';
+      whereLanguage = config.select[0].aggConditionLanguage ?? 'lucene';
```

Comment:
I think this is okay for now. Adding a note for future reference: only fields like ServerName or ResourceAttribute should be valid for cross-filtering.